app/src/main/java/com/example/musictoned/util/LocalStorage.kt

```diff
@@ -65,6 +66,15 @@ object LocalStorage {
 
     }
 
+    //Deletes the old workouts.json file and writes a new one with the updated 'allWorkouts' ArrayList
+    fun deleteWorkoutFile(){
+
+        val file:File = app.applicationContext.getFileStreamPath(workoutsFilename)
+        file.delete()
```

Comment:
`writeWorkouts` should overwrite the file, at least according to the comment on line 59, so we might not need to explicitly delete it.

Reply:
That's right, thanks. I'll update that later today.

Reply:
I fixed this issue